cob_hardware_config/cob4-4/config/diagnostics_analyzers.yaml

```diff
@@ -53,7 +53,12 @@ analyzers:
       base:
         type: diagnostic_aggregator/GenericAnalyzer
         path: Base
-        contains: 'base_controller'
+        contains: 'base'
```

Comment:
but the PR https://github.com/ipa320/cob_robots/pull/337 is still open, right? which PR should include the changes?